shared/settings/notifications/index.desktop.js

```diff
@@ -7,10 +7,10 @@ import type {Props} from './index'
 
 function Notifications (props: Props) {
   return (
-    <Box style={{...globalStyles.flexBoxColumn}}>
+    <Box style={{...globalStyles.flexBoxColumn, padding: globalMargins.medium}}>
       <Text type='Header' style={{marginTop: globalMargins.xlarge}}>Email me:</Text>
       <Box style={globalStyles.flexBoxColumn}>
-        {props.settings.map(s => (
+        {!!props.settings && props.settings.map(s => (
```

Comment:
Perhaps instead of hiding the checkboxes until the data loads, they could be visible disabled, so the checkbox ui doesn't pop in?